File: python/raindrops/raindrops.py
def convert(number):
    res = ""
    
    if number % 3 == 0:
        res += "Pling"
    if number % 5 == 0:
        res += "Plang"
    if number % 7 == 0:
        res += "Plong"

    if len(res) > 0:
        return res

    return f"{number}"

File: fsharp/raindrops/Raindrops.fs
module Raindrops

let convert (number: int) =
    [(3, "Pling"); (5, "Plang"); (7, "Plong")]
    |> List.filter (fun (d, _) -> number % d = 0)
    |> List.map snd
    |> function
       | [] -> string number
       | sounds -> String.concat "" sounds